kairos-test-utils/src/lib.rs

```diff
@@ -1 +1,3 @@
+#![feature(async_closure)]
```

Comment:
```suggestion
```

I removed this line, and it successfully compiles with the latest stable. Are you sure this feature is necessary?